src/calendars/general.ts

```diff
@@ -845,7 +860,7 @@ const dates = async (config: Config): Promise<Array<RomcalDateItemInput>> => {
     },
     {
       key: 'saintBridgetOfSwedenReligious',
-      type: TypesEnum.FEAST,
+      type: TypesEnum.OPT_MEMORIAL,
       date: dayjs.utc(`${year}-7-23`),
```

Comment:
Are you sure of that? Would not be just for Europe?

Reply:
According the the [WIKI](https://en.wikipedia.org/wiki/General_Roman_Calendar#July), in `general.ts`, it is a mere optional memorials. In Europe, the celebration’s rank is raised to feast, as St Bridget of Sweden is patron of Europe.

Is your question related to something else?

Reply:
Yes, sorry I misread your correction and saw the opposite.